src/chess_teacher_stockfish/main.py

```diff
@@ -84,8 +98,17 @@ class AnalyzeMoveResponse(CamelCaseAliasBaseModel):
     absolute_evaluation_change: float
 
 
-@app.get("/")
-def get_top_three_moves() -> DataResponse[TopThreeMovesResponse]:
+@app.get("/robots.txt", response_class=PlainTextResponse)
+def robots():
+    """Opens and prints the robots.txt file"""
+
+    with open(
+        os.path.join(os.path.dirname(__file__), "robots.txt"), encoding="utf-8"
+    ) as f:
+        return f.read()
+
+
+def get_top_three_moves() -> List[TopMove]:
```

Comment:
We actually don't need this function, it can be removed and the logic moved to `evaluate_position`, so that the current FEN state is applied before getting the top three moves from that position